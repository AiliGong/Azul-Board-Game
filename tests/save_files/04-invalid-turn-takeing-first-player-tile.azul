BYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRUL
A
B
C
D
turn 2 B 3
turn 3 L 3
