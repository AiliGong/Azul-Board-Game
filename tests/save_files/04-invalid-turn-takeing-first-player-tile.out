Received invalid turn specification: turn 0 F 0
